app/controllers/render-tree.js

```diff
@@ -2,16 +2,44 @@ import Ember from "ember";
 import escapeRegExp from "ember-inspector/utils/escape-reg-exp";
 import debounceComputed from "ember-inspector/computed/debounce";
 
-const { computed, isEmpty, Controller, inject: { controller } } = Ember;
+const { computed, isEmpty, Controller, inject: { controller, service } } = Ember;
 const { and, equal, filter } = computed;
 const get = Ember.get;
+let LOCAL_STORAGE_SUPPORTED;
+try {
+  LOCAL_STORAGE_SUPPORTED = !!window.localStorage;
+} catch (e) {
+  // Security setting in chrome that disables storage for third party
+  // throws an error when `localStorage` is accessed.
+  LOCAL_STORAGE_SUPPORTED = false;
+}
```

Comment:
@btecu the inspector works with Safari if you use the bookmarklet. Thanks for the suggestion.